components/metrics/metrics_log.cc

```diff
@@ -329,14 +329,16 @@ void MetricsLog::TruncateEvents() {
         internal::kUserActionEventLimit,
         uma_proto_.user_action_event_size() - internal::kUserActionEventLimit);
   }
-
+  // Omnibox proto removed for binary size reasons: b/290819695.
```

Comment:
Minor style comment - I think it would be more clear that the comment is describing the `#if` directive if the comment were aligned with it (no indentation). Does the formatter complain about that? If so, I would probably prefer to have the comment on the line directly below the `#if` directive, aligned with the code that follows it - that's what I've usually done in this case and I think also reads clearly.